README.md

```diff
@@ -1,2 +1,53 @@
-# sitecore-listexporter
-Allows you export the last known campaign ID of a contact in a list.
+﻿# sitecore-campaignlistexporter
+![Trigerfish Logo](https://avatars.githubusercontent.com/u/78132139?v=4)
+
+Allows you to export the last known campaign ID of a contact in a list.
```

Comment:
I think reword this:

## What is it?

This is a Sitecore List Manager extension module that allows you to export the last known campaign name of a contact in a list.

## Why?

Our use-case was when a user unsubscribes from all EXM campaigns, we wanted to know the last one they received before unsubscribing.

Reply:
Thanks Mark , Nice one , Updated